tensorflow/tools/docker/Dockerfile

```diff
@@ -44,7 +45,7 @@ RUN pip --no-cache-dir install \
 
 # Install TensorFlow CPU version from central repo
 RUN pip --no-cache-dir install \
-    http://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.0.0-cp27-none-linux_x86_64.whl
+    http://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-1.3.0rc2-cp27-none-linux_x86_64.whl
```

Comment:
These lines are automatically edited during the build processed described at:
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/docker#rebuilding-the-containers

Reply:
OK, you mean that 0.0.0 auto pointed to the latest version of cp2?

Reply:
No. I meant please do not change this line :)